topo/node/cptx/cptx_test.go

```diff
@@ -129,9 +130,15 @@ func TestConfigPush(t *testing.T) {
 			}
 			defer fp.Close()
 
+			/* Using scrapl_cfg_testing results in an EOF error when config includes comments.
+			 * We remove the first line of the sample config, which holds the plaintext password.
+			 * Comments in config files are not problematic when using kne_cli. */
+			fbuf := bufio.NewReader(fp)
+			_, _ = fbuf.ReadBytes('\n')
```

Comment:
even though an err here is almost impossible, it should still be checked and t.Fatal if the err is non-nil